### DISCUSSION TITLE
Running from source code, where to add certificate generated from EJBCA in system so SignServer can access it?

### DISCUSSION THREAD

**Discussion by r-shafi** [Keyfactor/signserver-ce#96] [Q&A]
Hello, I've followed documentations and have been able to build and deploy signserver from source code, but now I am unable to access the adminweb of signserver because it is not finding the ManagementCA.crt, and I do not know where to place it for it to be accessible.

Regards,
Shafi Rayhan.

**Comment by netmackan** [comment]
I will convert your issue into a Discussions thread as this is more about getting help and you will probably have more answers then.

**Reply by netmackan** [reply]
Hi Shafi,

You need to import the CA certificate of the CA that issues your admin certificate as trusted into the trust store used by the application server.
Following the installation guide you would typically have something like /opt/wildfly/standalone/configuration/keystores/truststore.jks.

To import it into the truststore you can use the Java KeyTool with something like:
```
keytool -importcert -keystore /opt/wildfly/standalone/configuration/keystores/truststore.jks -file /path/to/ManagementCA.pem -storepass changeit -alias MyManagementCA
```
